### PR TITLE
docs: Add description about img in artifact-visualization.md

### PR DESCRIPTION
**Pull request by juijeong8324**
### Motivation

image
It is difficult to know that img can link to play demo.

### Modifications
image
Therefore I add description of img.

### Verification
.

### CODE REVIEW

**Review by terrytangyuan — APPROVED** [review]

File: docs/artifact-visualization.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Use cases:
 * Visualizing end results of ML pipeline runs.
 * Debugging workflows where visual artifacts are the most helpful.
 
+> **Click image to play demo video**      
 [![Demo](https://img.youtube.com/vi/whoRfYY9Fhk/0.jpg)](https://youtu.be/whoRfYY9Fhk)
```
Comment (agilgur5) [inline docs/artifact-visualization.md:14]:
```suggestion
> Click the image below to play a demo video

[![Demo](https://img.youtube.com/vi/whoRfYY9Fhk/0.jpg)](https://youtu.be/whoRfYY9Fhk)
```

a new line is needed here as in the current screenshot, the image is _within_ the blockquote.
bold on top of the blockquote is not necessary, IMO.
also removed the trailing spaces that are causing lint to fail

**Review by agilgur5 — COMMENTED** [review]
I didn't even know this was a video, nice change!

I am thinking though that there are better, more conventional ways of indicating that it is a video:
1. Embed the video directly in the page
1. Use an image with a play button in the middle (or otherwise overlay such a play button)

File: docs/artifact-visualization.md (hunk 1 above)
Comment (juijeong8324) [reply docs/artifact-visualization.md:14]: Thank you for the helpful advice!!!

**Comment by juijeong8324** [comment]
> I didn't even know this was a video, nice change!
> 
> I am thinking though that there are better, more conventional ways of indicating that it is a video:
> 
> 1. Embed the video directly in the page
> 2. Use an image with a play button in the middle (or otherwise overlay such a play button)

**Comment by juijeong8324** [comment]
> I didn't even know this was a video, nice change!
> 
> I am thinking though that there are better, more conventional ways of indicating that it is a video:
> 
> 1. Embed the video directly in the page
> 2. Use an image with a play button in the middle (or otherwise overlay such a play button)

I contemplated the idea of generating and incorporating a distinct image, but it seemed challenging to upload a new image to a public repository and guarantee its credibility. Consequently, I opted to add a caption at the bottom of the image!

**Review by agilgur5 — APPROVED** [review]
> Consequently, I opted to add a caption at the bottom of the image!

This is still an improvement over what it was before, so I'm ok with this.

For other readers/reviewers, see the [rendered version](https://github.com/argoproj/argo-workflows/blob/1d3e87581ccc302b388635bd7838c7083ef2ba96/docs/artifact-visualization.md#artifact-visualization)

**Comment by agilgur5** [comment]
> I contemplated the idea of generating and incorporating a distinct image, but it seemed challenging to upload a new image to a public repository and guarantee its credibility.

The [docs/assets/](https://github.com/argoproj/argo-workflows/tree/v3.5.0-rc1/docs/assets) directory has many images already, so one could be added there.

**Review by agilgur5 — APPROVED** [review]
Nice! LGTM, thanks for iterating on this!

**Comment by juijeong8324** [comment]
Actually I have tried to use `<iframe>` tag to embed the video directly, it can't apply. 
However I find out that `<iframe>` tag is not supported in Github Markdown. ([Disallowed Raw HTML (extension)](https://github.github.com/gfm/#disallowed-raw-html-extension-) or [Stack Overflow](https://stackoverflow.com/questions/48402823/embed-openstreetmap-iframe-in-github-markdown)) If you know how to embed the video in Github markdown, please let me know.

**Comment by agilgur5** [comment]
Yea it's not allowed in GH Markdown (no interactive elements basically), but I would think it would work on the rendered docs site. Did it not work on the docs site?

**Pull request merged**